src/dune_rules/simple_rules.ml

```diff
@@ -232,22 +232,13 @@ let alias sctx ?extra_bindings ~dir ~expander (alias_conf : Alias_conf.t) =
       Rules.Produce.Alias.add_deps alias ?loc builder
     | Some (action_loc, action) ->
       let action =
-        let builder, expander, sandbox =
-          Dep_conf_eval.named ~expander alias_conf.deps
```

Comment:
Isn't this line needed so that names in `alias_conf.deps` are available as `%` forms?